_positions/visual-designer.md

```diff
@@ -27,3 +27,5 @@ Visual designers at 18F craft highly usable and beautiful web-based experiences
 - Regularly participate in group critiques, serving as both “critic” and the “critiqued.”
 - Scour the web for applicable new design solutions and share findings with the team.
 - Support the 18F brand by contributing to in-house projects.
+
+##[Apply to join 18F]({{ site.baseurl }}/pages/apply.html)
```

Comment:
In the end state, we don't want two application streams, we only want people to be able to apply to specific roles. Part of the formatting/copy choices here were to be able to just launch the Lever app with a single role to test but make the updates to the other pages so that formatting would be somewhat consistent.